### DISCUSSION TITLE
How to run them？

### DISCUSSION THREAD

**Discussion by longmarchw** [DrewKestell/BloogBot#20] [General]
Once compiled, how do I run them, and on what operating system do they work?
How do I create a database file? Not sure, where can I get support?

**Comment by DrewKestell** [comment]
Windows is the only supported operating system.

To run the bot, follow the instructions in [Getting Started](https://discord.gg/S4tvykaGcJ). Out of the box, it'll create a SQLite database for you on your local machine and bootstrap all the necessary tables.

Note that this is a free open source project and as such I don't really provide "support". But you should join the [public Discord server](https://discord.gg/S4tvykaGcJ) if you need help with anything.

**Comment by pablovera2735** [comment]
Hello, I wanted to ask if there's any way to run the bot on Linux, even using Wine/Proton, Docker, or some other unofficial method.

I have a relative who plays WoW on Bazzite OS, and we'd like to know if you plan to provide Linux support in the future, or if it's simply not feasible at the moment.

I understand the project is free and open source, so I'm not expecting formal technical support; I just wanted to confirm if there's any possibility or plan for Linux.

Thanks!

**Reply by longmarchw** [reply]
It seems impossible. it  is only a free open source project. The author hasn't updated in a long time...

**Reply by DrewKestell** [reply]
I'm not saying it's impossible, but as I mentioned above it's not supported and I'm not going to spend time on adding that functionality. If you want to get it working, I'd be happy to merge the change back into the repo.
